Tests/OutlineViewTests/OutlineViewDataSourceTests.swift

```diff
@@ -13,16 +13,17 @@ class OutlineViewDataSourceTests: XCTestCase {
         TestItem(id: 3, children: [TestItem(id: 4, children: nil)]),
     ]
     .map { OutlineViewItem(value: $0, children: \TestItem.children) }
+    let itemChildren: ChildSource<[TestItem]> = .keyPath(\.children)
 
     let outlineView = NSOutlineView()
 
     func testInit() {
-        let dataSource = OutlineViewDataSource(items: items)
+        let dataSource = OutlineViewDataSource(items: items, childSource: .keyPath(\.children))
```

Comment:
The tests no longer build because the `Drop` generic is not specified. You can consider declaring a `typealias` in the `OutlineViewDataSourceTests` class.

```swift
typealias DataSource = OutlineViewDataSource<[TestItem], NoDropReceiver<TestItem>>

func testInit() {
    let dataSource = DataSource(items: items, childSource: .keyPath(\.children))
    [...]
}
```

Reply:
Whoops! Fixed: https://github.com/Sameesunkaria/OutlineView/pull/10/commits/2a4af9d9a0c5ef022b58946da454419c9b3f61d3